draft-ietf-quic-multipath.md

```diff
@@ -1180,31 +1180,79 @@ but has not provided a corresponding connection ID for the path ID
 
 # Implementation Considerations {#impl-consideration}
 
-## Number Spaces
+This section provides informational guidance for implementors.
 
-As stated in {{introduction}}, when the multipath extension is negotiated, each
+## CID Changes, Migration, and NAT Rebindings {#migration}
+
+With the multipath extension each
 path uses a separate packet number space.
 This is a major difference from
 {{QUIC-TRANSPORT}}, which only defines three number spaces (Initial,
 Handshake and Application packets).
 
 The relation between packet number spaces and paths is fixed.
 Connection IDs are separately allocated for each Path ID.
-Rotating the connection ID on a path does not change the Path ID.
-NAT rebinding, though it changes the 4-tuple of the path,
-also does not change the path identifier.
-The packet number space does not change when connection ID
-rotation happens within a given Path ID.
+Rotating the connection ID on a path does not change the Path ID
+nor packet number space.
+
+While endpoints assign a connection ID to a specific sending 4-tuple,
+networks events such as NAT rebinding may make the packet's receiver
+observe a different 4-tuple. Though it changes the 4-tuple of the path,
+also does not change connection ID and as such also not the path identifier.
```

Comment:
This sentence is hard to parse, but I think that I understand the underlying meaning

```suggestion
the connection ID remains the same, and so the related path identifier.
```